.gitignore

```diff
@@ -12,3 +12,5 @@ versions.json
 .eslintcache
 .cache
 **/node_modules
+/tests/performance/eslintrc.yml
+/tests/performance/eslint/
```

Comment:
Those files are created in `npm run perf` command.
So this is to avoid to make extra diff after the command.